pyam/iiasa.py

```diff
@@ -320,11 +326,15 @@ def query(self, **kwargs):
         r = requests.post(url, headers=headers, data=data)
         _check_response(r)
         # refactor returned json object to be castable to an IamDataFrame
-        df = (
-            pd.read_json(r.content, orient='records')
-            .drop(columns='runId')
-            .rename(columns={'time': 'subannual'})
-        )
+        df = pd.read_json(r.content, orient='records')
+        columns = ['model', 'scenario', 'variable', 'unit',
+                   'region', 'year', 'value', 'time', 'meta',
+                   'runId', 'version']
+        df = pd.DataFrame(data=df, columns=columns)
```

Comment:
this seems a bit brittle to me - e.g., what if the rest api adds additional columns to the return obj at some future point in time?

Would suggest something more like:

```
columns = ['model', 'scenario', 'variable', 'unit',
                   'region', 'year', 'value', 'time', 'meta',
                   'version']
df = pd.read_json(r.content, orient='records')
drop = list(set(df.columns) - set(columns))
df.drop(columns=drop, inplace=True)
df.fillna({'meta': 0}, inplace=True)
df.rename(columns={'time': 'subannual'}, inplace=True)
```

Reply:
thanks